konflux_scripts/parse-snapshot.py

```diff
@@ -58,7 +58,12 @@ def main() -> None:
     snapshot: Snapshot = Snapshot.model_validate_json(snapshot_str)
     ret = []
     for component in snapshot.components:
-        component_name = os.environ.get("BONFIRE_COMPONENT_NAME", component.name)
+        try:
+            component_name = os.environ['BONFIRE_COMPONENT_NAME']
+            if component_name == "":
+                component_name = component.name
+        except KeyError:
+            component_name = component.name
```

Comment:
```suggestion
            component_name = os.environ.get('BONFIRE_COMPONENT_NAME', component.name)
```

Reply:
The issue here is that it won't fail with the `""`, which is the value by default